bigbluebutton-html5/imports/ui/core/hooks/createUseSubscription.ts

```diff
@@ -45,7 +74,8 @@ function createUseSubscription<T>(
           },
         });
       return () => apolloSubscription.unsubscribe();
-    }, []);
+    }, [JSON.stringify(variables)]);
```

Comment:
As the `variables` comes from useMemo producing a JSON, it seems this `JSON.stringify` is redundant.